addons/sale_management/models/sale_order_template_line.py

```diff
@@ -39,10 +39,7 @@ class SaleOrderTemplateLine(models.Model):
 
     name = fields.Text(
         string="Description",
-        compute='_compute_name',
-        store=True, readonly=False, precompute=True,
-        required=True,
-        translate=True)
```

Comment:
not translatable anymore ? :thinking:

Reply:
It seems there has been a mistake! I will add the `translate=True` right away. Thanks!